assets/src/application/ui/display/confirm/useConfirm.tsx

```diff
@@ -1,12 +1,15 @@
-import React, { useCallback, useMemo } from 'react';
-import { Button, useDisclosure } from '@chakra-ui/core';
+import React, { useCallback } from 'react';
+import { useDisclosure } from '@chakra-ui/core';
 import { __ } from '@wordpress/i18n';
 
 import { EspressoButton } from '@application/ui/input';
 import { AlertDialog } from '@infraUI/display';
+import { Button } from '@infraUI/inputs';
 
 import { ConfirmProps } from './types';
 
+// TODO: fix file name and add suport for EspressoIconButton e.g. for TPC delete price button
```

Comment:
it was initially `Confirm`, but after this short comment: https://github.com/eventespresso/event-espresso-core/pull/2682#discussion_r406828007
it was renamed/ repurpose to `useConfirm`